src/pytest_adf/pytest_adf.py

```diff
@@ -4,12 +4,28 @@
 import os
 import time
 import logging
+import json
+from datetime import timedelta
+from datetime import datetime
+from pytest_adf.pytest_adf_enums import BlobEventType
+from pytest_adf.pytest_adf_enums import TriggerRunDataAPIAction
 from azure.identity import ClientSecretCredential, AzureCliCredential
 from azure.mgmt.datafactory import DataFactoryManagementClient
+from azure.mgmt.datafactory.models._models import RunQueryFilter
+from azure.mgmt.datafactory.models._models import RunQueryOrderBy
+from azure.mgmt.datafactory.models._models import RunFilterParameters
+from azure.mgmt.datafactory.models._data_factory_management_client_enums import RunQueryOrderByField
+from azure.mgmt.datafactory.models._data_factory_management_client_enums import RunQueryOrder
+from azure.mgmt.datafactory.models._data_factory_management_client_enums import RunQueryFilterOperand
+from azure.mgmt.datafactory.models._data_factory_management_client_enums import RunQueryFilterOperator
+from azure.storage.blob import BlobServiceClient
 
 LOG = logging.getLogger(__name__)
 
-OPTIONAL_ARGS = ["AZ_SERVICE_PRINCIPAL_ID", "AZ_SERVICE_PRINCIPAL_SECRET"]
+OPTIONAL_ARGS = ["AZ_SERVICE_PRINCIPAL_ID", "AZ_SERVICE_PRINCIPAL_SECRET", "AZ_STORAGE_ACCOUNT_NAME", "AZ_STORAGE_ACCOUNT_KEY"]
+OPTIONAL_STORAGE_ACCOUNT_ARGS = ["AZ_STORAGE_ACCOUNT_NAME", "AZ_STORAGE_ACCOUNT_KEY"]
+FETCHING_TRIGGER_RUNS_MAX_TIME_IN_SECONDS = 180
```

Comment:
Add some comments about these constants. I was initially confused on the difference between the two and had to go through the code to figure it out :). 

How about this:
```python
# Determines the ADF query time range to search for the corresponding trigger ADF pipeline starting from when the blob file is uploaded/deleted. 
# For example, if file is uploaded/deleted at 01:00, this will search for the triggered pipelines between 01:00 and 01:03.
FETCHING_TRIGGER_RUNS_TIME_RANGE_IN_MINUTES = 180

# Determines the maximum amount of time pytest-adf will look for the triggered pipeline run within the FETCHING_TRIGGER_RUNS_TIME_RANGE_IN_MINUTES
FETCHING_TRIGGER_RUNS_MAX_TIME_IN_SECONDS = 3
```